web/tests/people.test.cjs

```diff
@@ -1711,11 +1715,23 @@ test_people("fetch_users", async ({override}) => {
         assert.ok(data.user_ids.includes("16"));
         assert.ok(!data.user_ids.includes("42"));
         assert.ok(!data.user_ids.includes("17"));
+
+        // While this get request is in-flight, request for the
+        // a subset of the users again and check if it resolved.
+        const promise = people.get_or_fetch_users_from_ids([15]);
         success({
             members: users_in_response,
             result: "success",
             msg: "",
         });
+        // We know another `get` request call was not made since
+        // `assert.ok(data.user_ids.includes("16"));` would have failed
+        // above if that were the case.
+        promise.then((users) => {
```

Comment:
Use `await`. (If you stick a random `.then` in a test, it likely won’t run until after the test has been cleaned up, potentially causing weird issues with future tests that are hard to debug.)